ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -252,6 +252,8 @@ function suite() {
 echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_TEST_START"
 trap 'echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_TEST_END"' EXIT
 
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.8-kube-1.22-api-removals-in-4.9":"true"}}' --type=merge || echo 'failed to ack the 4.9 Kube v1beta1 removals; possibly API-server issue, or a pre-4.8 release image'
```

Comment:
this is the best place we can stitch this in?  not in some upgrade or install step?

Reply:
We don't want it in an update step, because the update jobs are actually ok (they don't sit on a 4.8 release for long enough for the alert to move from pending to firing).  And the job I linked from my initial comment / commit message was `periodic-ci-openshift-release-master-nightly-4.8-e2e-aws-serial`, so not an update job.  Also, openshift/origin#26446 (which we want too) is a nice, central fix for update jobs.

I don't mind putting it at the end of an install step, but we have a few install steps in different workflows that will all end up calling the test suite here.  We might also have a few different test steps besides this one that would sit on an 4.8 release for long enough and be bothered by the firing alert (OSD CI?).  But it's hard for me to imagine more of those vs. install steps.  In any case, the only failure caused by this alert due to a lack-of-ack on 4.8 that I'm aware of today uses this test step, and I know all 4.8 installs (regardless of platform or installer-provided vs. user-provided infra) will be exposed, and I'm not aware of other test steps besides this one that would care.

Reply:
good enough for me